test/functional/tests/failover_standby/test_cache_activation_perf.py

```diff
@@ -0,0 +1,142 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+from datetime import timedelta, datetime
+
+import pytest
+
+from core.test_run import TestRun
+from test_tools.dd import Dd
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+from test_utils.filesystem.file import File
+from test_utils.os_utils import Udev
+from test_utils.size import Size, Unit
+from storage_devices.disk import DiskType, DiskTypeSet
+from api.cas import casadm, dmesg
+from api.cas.cache_config import CleaningPolicy, CacheMode, CacheLineSize, SeqCutOffPolicy
+
+activation_time_threshold = timedelta(seconds=10)
+test_drive_size = Size(800, Unit.GibiByte)
+
+
+@pytest.mark.require_disk("cache_1", DiskTypeSet([DiskType.nand, DiskType.optane]))
+@pytest.mark.require_disk("cache_2", DiskTypeSet([DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeSet([DiskType.hdd, DiskType.hdd4k]))
+def test_cache_activation_time():
+    """
+    title: Measure standby cache activation time
+    description: Determine if the activation of a standby cache completes within
+                 a predefined time limit.
+    pass_criteria:
+      - The activation completes within the time limit
+    """
+    with TestRun.step("Setting up two cache devices, 800GB each"):
```

Comment:
How about adding an explicit check to verify that test has devices of a sufficient size?